<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshCheckTabWidget</class>
 <widget class="QWidget" name="MeshCheckTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="labelMesh">
       <property name="text">
        <string>Mesh to fix:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMesh"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxNonTriangles">
       <property name="text">
        <string>Non-Triangles</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxConnectivity">
       <property name="text">
        <string>Connectivity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBoxDegeneratedElements">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Degenerated Elements</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxOrientation">
       <property name="text">
        <string>Triangle orientation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxBorders">
       <property name="text">
        <string>Borders</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxInconsistency">
       <property name="text">
        <string>Inconsistency</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonCheckMesh">
     <property name="text">
      <string>Check Mesh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelOutput">
     <property name="text">
      <string>Might take some time and memory.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowserOutput"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
